### PR TITLE
Add liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc

### PR DESCRIPTION
**Pull request by yeekangc**
Blog post about updates to Liberty Tools and Liberty build plugins.

For #3184

### CODE REVIEW

**Comment by cherylking** [comment]
Are the first two links for the blogs correct? I was not able to navigate to either one.

**Comment by chirp1** [comment]
@yeekangc Hi YK, Thanks for alerting us to this blog post issue. I talked with Ram. He is going to do the review for the ID team.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Learn more about link:/docs/latest/develop-liberty-tools.html[Liberty Tools] and download them from the different IDE marketplaces:
+
+* link:https://marketplace.eclipse.org/content/liberty-tools[Liberty Tools for Eclipse IDE] 
+* link:https://marketplace.visualstudio.com/items?itemName=Open-Liberty.liberty-dev-vscode-ext[Liberty Tools for Visual Studio Code] 
+* link:https://plugins.jetbrains.com/plugin/14856-liberty-tools[Liberty Tools for IntelliJ IDEA] 
+
+Do not hesitate to share your feedback and what you like to see in future releases via issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].
```
Comment (scottkurz) [inline posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]:
```suggestion
Do not hesitate to share your feedback and what you would like to see in future releases via issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].
```

Comment (scottkurz) [reply posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: Add a word?

**Comment by ramkumar-k-9286** [comment]
@yeekangc @chirp1 

Hi,
Please find my review comments below:

Peer Review
---

Liberty Tools 23.0.6 are now available for Java developers to use to build cloud-native applications with Liberty.
->
Liberty Tools 23.0.6 are now available **for use for Java developers to build** cloud-native applications with Liberty.

---

With support for popular integrated development environments (IDEs) like Eclipse IDE, Visual Studio Code, and IntelliJ IDEA, Liberty Tools enable developers to focus on their code and rapidly iterate via integration with Liberty's link:/docs/latest/development-mode.html[dev mode] using their IDE of choice.
->
Liberty Tools support popular integrated development environments (IDEs) such as Eclipse, Visual Studio Code, and IntelliJ IDEA, allowing developers to focus on their code and rapidly iterate by using Liberty's link:/docs/latest/development-mode.html[dev mode] in their preferred IDE.

---

Moreover, Liberty Tools provide editing assistance for developers to easily make changes to Liberty configuration files and coding assistance for writing applications using Jakarta EE (9.x and later) and MicroProfile (3.x and later) APIs.
->
Moreover, Liberty Tools provide editing assistance for developers to easily make changes to **the** Liberty configuration **files,** and coding assistance for writing applications **that use** Jakarta EE (9.x and later) and MicroProfile (3.x and later) APIs.

---

Learn more about link:/docs/latest/develop-liberty-tools.html[Liberty Tools] and download them from the different IDE marketplaces:
->
Learn more about link:/docs/latest/develop-liberty-tools.html[Liberty Tools] and download them from different IDE marketplaces:

---

Do not hesitate to share your feedback and what you like to see in future releases via issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].
->
Do not hesitate to share your feedback and what you **would** like to see in future releases via issues in the GitHub repositories for link:https://github.com/OpenLiberty/liberty-tools-eclipse/issues/new[Liberty Tools for Eclipse IDE], link:https://github.com/OpenLiberty/liberty-tools-vscode/issues/new[Liberty Tools for Visual Studio Code], and link:https://github.com/OpenLiberty/liberty-tools-intellij/issues/new[Liberty Tools for IntelliJ IDEA].

---

== Build, package and run easily with Liberty Maven & Gradle plugins

It's always best to use the latest versions of these build plugins to ensure you have the latest fixes and enhancements.  For example, starting with the 3.6 release, the Liberty Gradle Plugin supports running with Gradle 8.x.  Find out more: link:https://github.com/OpenLiberty/ci.maven/releases[Liberty Maven Plugin releases] and link:https://github.com/OpenLiberty/ci.gradle/releases[Liberty Gradle Plugin releases].

To use Liberty Maven Plugin 3.8.2, in your Maven build file, specify:
->
== Build, package, and run easily with Liberty Maven & Gradle plugins

It is always recommended to use the latest versions of these build plugins to ensure you have the most recent fixes and enhancements. For example, starting with the 3.6 release, the Liberty Gradle Plugin supports running with Gradle 8.x. For more information, see link:https://github.com/OpenLiberty/ci.maven/releases[Liberty Maven Plugin releases] and link:https://github.com/OpenLiberty/ci.gradle/releases[Liberty Gradle Plugin releases].

To use Liberty Maven Plugin 3.8.2, specify the following in your Maven build file:

---

To use Liberty Gradle Plugin 3.6.2, in your Gradle build file, specify:
->
To use Liberty Gradle Plugin 3.6.2, specify the following in your Gradle build file:

---

Like the Open Liberty runtime itself, all of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin are open sourced and openly developed under the link:https://github.com/OpenLiberty[OpenLiberty] organization on GitHub.  You are welcome to contribute directly and make Liberty even better!
->
Like the Open Liberty runtime itself, all of Liberty Tools, Liberty Maven Plugin, and Liberty Gradle Plugin are open sourced and developed under the link:https://github.com/OpenLiberty[OpenLiberty] organization on GitHub. You are welcome to contribute directly and make Liberty even better!

( are **open sourced** and **openly** developed under -> **open** seems to be kind of repetitive), also, additional space before 'You are welcome to contribute'.

---

Take a look at code, grab an issue, and send in a pull request:
->
Take a look at **the** code, grab an issue, and send in a pull request:

---

**Comment by yeekangc** [comment]
> Are the first two links for the blogs correct? I was not able to navigate to either one.

That should be how relative links are specified.  Will double check once we merged into draft and have a build.  Thanks.

File: posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc (hunk 1 above)
Comment (yeekangc) [reply posts/2023-07-10-liberty-ide-tools-maven-gradle-plugins-for-java-developers.adoc]: What a miss!

**Pull request merged**